### PR TITLE
Chore 49 cleanup react styles

### PR DESCRIPTION
**Pull request by winkerVSbecks**
Cleanup Styles
- Removed the column component.
- Updated container to basscss v8 syntax and added more props.
- Removed styles and classnames being passed into components.
- Introduced a classnames library to manage stateful classes.

### CODE REVIEW

File: src/components/alert.js [hunk 1]
```diff
@@ -1,19 +1,27 @@
...
-  const visibleClass = isVisible ? 'block' : 'hide';
+const Alert = ({
+  children,
+  isVisible,
+  status = 'info',
+}) => {
+  const alertClasses = classNames(['p2', 'bold'], {
+    block: isVisible,
+    hide: !isVisible,
+    [statusClasses[status]]: true,
```
Comment (SethDavenport) [inline src/components/alert.js]: :+1:

**Comment by SethDavenport** [comment]
:shipit:

Thanks for cleaning up all those `style` and `className` props, this looks great.  You're going to hate me but feel like duplicating it in the typescript-react-starter :) ?

@mattmcfad : this PR may interest you with respect to the work you're doing on the ng2 one.

**Comment by winkerVSbecks** [comment]
@SethDavenport yup, was going to move on to that next.

**Comment by SethDavenport** [comment]
Awesome, thanks.

**Pull request merged**